docs/faq.md

```diff
@@ -103,14 +103,17 @@ Training, validation, and test splits for the non-enzymatic dataset are availabl
 
 **How do I know which model to use after training Casanovo?**
 
-By default, Casanovo saves a snapshot of the model weights after every 50,000 training steps.
+By default, Casanovo runs model validation every 50,000 training steps.
 Note that the number of samples that are processed during a single training step depends on the batch size.
 Therefore, the default training batch size of 32 corresponds to saving a model snapshot after every 1.6 million training samples.
-You can optionally modify the snapshot (and validation) frequency in the [config file](https://github.com/Noble-Lab/casanovo/blob/main/casanovo/config.yaml) (parameter `val_check_interval`), depending on your dataset size.
-Note that taking very frequent model snapshots will result in slower training time because Casanovo will evaluate its performance on the validation data for every snapshot.
-
-When saving a model snapshot, Casanovo will use the validation data to compute performance measures (training loss, validation loss, amino acid precision, and peptide precision) and print this information to the console and log file.
-After your training job is finished, you can identify the model that achieves the maximum peptide and amino acid precision from the log file and use the corresponding model snapshot.
+You can optionally modify the validation run frequency in the [config file](https://github.com/Noble-Lab/casanovo/blob/main/casanovo/config.yaml) (parameter `val_check_interval`), depending on your dataset size.
+Note that running model validation very frequently will result in slower training time because Casanovo will evaluate its performance on the validation data for every validation check.
+
+When running model validation, Casanovo will use the validation data to compute performance measures (training loss, validation loss, amino acid precision, and peptide precision) and print this information to the console and log file.
```

Comment:
**suggestion:** I think that this paragraph should go first.
1. Which model to use -> determined by validation to select the best model.
2. Some info about how frequent validation occurs.